applications/vessel-history/public/locales/source/translations.json

```diff
@@ -84,16 +84,7 @@
     "fishingEvent": "Fishing in {{ regionName }}",
     "fishingGroup": "{{ count }} Fishing events in {{ regionName }}",
     "fishingGroup_plural": "{{ count }} Fishing events in {{ regionName }}",
-    "noDescription": "No description found",
-    "confidenceHelp": {
-      "help2": "Low confidence. Only a port stop OR gap was identified based on AIS transmission.",
-      "help3": "Medium confidence. A port entry and a port stop OR gap; OR a port stop OR gap followed by a port exit was identified based on AIS transmission.",
-      "help4": "High confidence. A port entry, AND port stop or port gap, AND a port exit was identified based on AIS transmission.",
-      "portEntry": "PORT ENTRY: vessel that was not in port gets within 3km of anchorage point",
-      "portExit": "PORT EXIT: vessel that was in port moves more than 4km from anchorage point",
-      "portGap": "PORT GAP: AIS gap > 4 hours; start is recorded 4 hours after the last message before the gap; end at next message after gap.",
-      "portStop": "PORT STOP: begin: speed < 0.2 knots; end: speed > 0.5 knots"
-    }
+    "noDescription": "No description found"
```

Comment:
Why removing the `"confidenceHelp"` from `source`?

Reply:
is duplicated :/